### PR TITLE
Fixes #37277 - Fix ACS randomly failing VCR tests

### PR DESCRIPTION
**Pull request by ianballou**
#### What are the changes introduced in this pull request?
Should fix the ruby tests run in the GH actions. There was ordering randomness with ACS paths.

#### Considerations taken when implementing this change?
Sorting during ACS creation / updating should have a very minimal affect on performance. Plus, it reduces randomness.

#### What are the testing steps for this pull request?
See that that tests all pass.

### CODE REVIEW

**Comment by ianballou** [comment]
test_bad_promote_out_of_sequence – Katello::Api::V2::ContentViewVersionsControllerTest is an unrelated failure, but I'll add the fix in here.

**Comment by ianballou** [comment]
Pipeline is green! @sjha4

**Review by jeremylenz — APPROVED** [review]
:green_apple: :green_apple:

File: app/services/katello/pulp3/alternate_content_source.rb [hunk 1]
```diff
@@ -108,7 +108,7 @@ def create
           if acs.content_type == ::Katello::Repository::FILE_TYPE && acs.subpaths.present?
             paths = insert_pulp_manifest!(paths)
           end
-          response = api.alternate_content_source_api.create(name: generate_backend_object_name, paths: paths,
+          response = api.alternate_content_source_api.create(name: generate_backend_object_name, paths: paths.sort,
```
Comment (sjha4) [inline app/services/katello/pulp3/alternate_content_source.rb:111]: Seems fair..Ack 👍🏼

**Review by sjha4 — APPROVED** [review]

**Pull request merged**